www/html/css/index2.css

```diff
@@ -1,3 +1,37 @@
+input.btn {
+  margin-top: 15px;
+}
+
+#copyright {
+  color: #fed136;
+  padding-left: 5px;
+  padding-right: 10px;
+}
+
+#lock {
+  color: #fed136;
+  padding-left: 10px;
+}
+
+table {
+  display: inline;
+}
+
+footer.container-fluid.center-block {
+  background-color: #262626;
+  height: 60px;
+  margin-top: 100;
+  padding-top: 15px;
+  text-align: center;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  height: 80px;
+  width: 100%;
+  overflow:hidden;
+  z-index: 9999;
+}
+
```

Comment:
add new code at the bottom of the file please. the top has general container rules that are nice to have at the top